docs/source/markdown/podman-run.1.md

```diff
@@ -608,6 +608,15 @@ Whether to disable OOM Killer for the container or not.
 
 Tune the host's OOM preferences for containers (accepts values from **-1000** to **1000**).
 
+**--override-arch**=*ARCH*
+Override the machine's default architecture of the image to be pulled. For example, `arm`.
```

Comment:
```suggestion
Override the machine's default architecture of the image to be run. For example, `arm`.
```

Reply:
You are not running an image, you are pulling it.